src/seedu/addressbook/common/UtilsTest.java

```diff
@@ -0,0 +1,38 @@
+package seedu.addressbook.common;
+
+import static org.junit.Assert.*;
+
+import java.util.ArrayList;
+import java.util.Collection;
+
+import org.junit.Test;
+
+public class UtilsTest {
+
+	@Test
+	public void testElementsAreUnique() {
```

Comment:
Indentation problem here? Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.